canonical-websites/www.ubuntu.com
# Update content and migrate /legal to v1 styling

## Done

Updated content at /legal/ubuntu-advantage/service-description

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/))
- Go to [http://0.0.0.0:8001/legal/ubuntu-advantage/service-description](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description)
- Check it matches the [copy doc](https://docs.google.com/document/d/1mrNrsC1WDf6QTg3nrVCEZS8vJzGz1fokvoFDVC4s_mI/edit#heading=h.juyf3k5nrdw8)


## Issue / Card

#1792 

## Demo

http://www.ubuntu.com-1792-service-desc-update.demo.haus/legal/ubuntu-advantage/service-description
